packages/react-core/src/demos/PrimaryDetail.md

```diff
@@ -205,7 +205,6 @@ class PrimaryDetailFullPage extends React.Component {
             <TextInput
               name="full-page-data-toolbar-input1"
               id="full-page-data-toolbar-input1"
-              type="search"
```

Comment:
@nicolethoen I imagine we'd want to keep these `type="search"` as well for TextInput components for now?

Reply:
added those back